source/fuzz/transformation_replace_id_with_synonym.cpp

```diff
@@ -140,6 +140,51 @@ bool TransformationReplaceIdWithSynonym::IsAgnosticToSignednessOfOperand(
     case SpvOpUGreaterThanEqual:
     case SpvOpSGreaterThanEqual:
       return true;
+
+    case SpvOpAtomicStore:
+    case SpvOpAtomicExchange:
+    case SpvOpAtomicIAdd:
+    case SpvOpAtomicISub:
+    case SpvOpAtomicSMin:
+    case SpvOpAtomicUMin:
+    case SpvOpAtomicSMax:
+    case SpvOpAtomicUMax:
+    case SpvOpAtomicAnd:
+    case SpvOpAtomicOr:
+    case SpvOpAtomicXor:
+    // TBD(To Be Discussed),
+    // https://github.com/KhronosGroup/SPIRV-Registry/blob/master/extensions/EXT/SPV_EXT_shader_atomic_float_add.asciidoc#modifications-to-the-spir-v-specification-version-15
+    case SpvOpAtomicFAddEXT:  // Capability (AtomicFloat32AddEXT,
+                              // AtomicFloat64AddEXT)
+      assert(use_in_operand_index != 0 && "Forbidden.");
+
+      if (opcode == SpvOpAtomicFAddEXT) {
+        // Would you like to check Capability here?
+      }
```

Comment:
You can include that instruction in the list of case labels, and there's no need to check for capability.

The reason is that whether signedness of operands matters is independent of the capability.

That's different from the situation with operands being OpConstant in your previous PR - there, the capability determined the rules. Here the capability only determines whether this instruction exists. If it doesn't exist we won't see it in this function, so that's OK :)

In short: you can just include it in the list of case labels, and then have your `return` statement handle all of them.

Reply:
Thanks for the clarification, done.